index.js

```diff
@@ -13,25 +13,36 @@ module.exports = {
 
   included() {
     let app = findHost(this);
+    let desiredComponents = [
+      'apacheconf',
+      'bash',
+      'css',
+      'http',
+      'javascript',
+      'json',
+      'json5',
+      'markup-templating',
+      'ruby',
+      'scss',
+      'yaml',
+      'typescript',
+      'diff',
+    ];
+    const existingComponents = app.options['ember-prism']?.components ?? [];
+    const newComponents = existingComponents.slice();
+    for (const component of desiredComponents) {
+      if (!existingComponents.includes(component)) {
+        newComponents.push(component);
+      }
+    }
+
     if (!app.options['ember-prism']) {
```

Comment:
Since we're adding an else block, let's switch the branches around to avoid "if not, else"

Reply:
That makes good sense.